Sources/StreamChatUI/ChatChannelList/ChatChannelListItemView.swift

```diff
@@ -8,14 +8,16 @@ import UIKit
 /// A `ChatChannelSwipeableListItemView` subclass view that shows channel information.
 public typealias ChatChannelListItemView = _ChatChannelListItemView<NoExtraData>
 
-/// A `ChatChannelSwipeableListItemView` subclass view that shows channel information.
-open class _ChatChannelListItemView<ExtraData: ExtraDataTypes>: _ChatChannelSwipeableListItemView<ExtraData> {
-    
+/// Base class of `ChatChannelListItemView`.
+open class _ChatChannelListItemViewBase<ExtraData: ExtraDataTypes>: _ChatChannelSwipeableListItemView<ExtraData> {
```

Comment:
I don't think we need the base class TBH. If the wrapper subclass won't call the `super` implementation of the `Customizable` methods, it will be just an empty shell.

Reply:
You are totally right, got rid of the base class.